user-guide/Advanced_Functionality/DataMiner_Systems/Adding_a_DMA/Adding_a_DMA_to_a_DMS_running_STaaS.md

```diff
@@ -4,43 +4,41 @@ uid: Adding_a_DMA_to_a_DMS_running_STaaS
 
 # Adding a DataMiner Agent to a DMS running STaaS
 
-Follow these steps below to connect a DMA to an existing DMS running STaaS:
+## Using a copy of the cloud persisted data
```

Comment:
Can you add information why you would choose to do it this way. Whats the advantage compared to local db procedure.

Reply:
If this is the preferred way, mention this as well

Reply:
Will do this once we add the flow with the local database again.